pkg/operator/cassandra/test/test.go

```diff
@@ -0,0 +1,32 @@
+package test
```

Comment:
Every single source code file needs to have the license/header.  Can you please ensure that all of these files have the header included? e.g. https://github.com/rook/rook/blob/master/tests/framework/installer/ceph_installer.go#L1-L15

Note that example yaml files do not need the license/header.